src/controllers/rivens.js

```diff
@@ -38,15 +38,11 @@ const groupRivenData = (cacheStrData) => {
 };
 
 platforms.forEach((platform) => {
-  const rCache = new Cache(
-    `https://n9e5v4d8.ssl.hwcdn.net/repos/weeklyRivens${platform.toUpperCase()}.json`,
-    604800000,
-    {
-      parser: groupRivenData,
-      logger,
-      delayStart: true,
-    }
-  );
+  const rCache = new Cache(`https://www.warframe.com/repos/weeklyRivens${platform.toUpperCase()}.json`, 604800000, {
```

Comment:
Not specifically related to the issue but as we are working with this, does the ttl in the cache (7 days) is in any way oriented with the time in which rivens are updated? For example in the worst case scenario the api would be outdated by 6 days 23 hours 59 minutes. Or am i missing something?

Reply:
Yeah, I can probably decrease this one to just a few hours and use the same cache we use elsewhere.